### ISSUE TITLE
CollectionView EmptyView not working

### ISSUE THREAD

**Issue by angelru** [dotnet/maui#7063]
### Description

It doesn't show the layout in EmptyView, I haven't been able to test it on iOS, on Windows it works fine

### Steps to Reproduce

  ```
<CollectionView Grid.Row="1" Margin="16,0,16,6" ItemsSource="{Binding Certificates}"
                        SelectionMode="Single"
                        ItemSizingStrategy="MeasureFirstItem"
                        SelectedItem="{Binding SelectedCertificateSigne}"
                        SelectionChangedCommand="{Binding CertificateSigneSelectionCommand}">
            <CollectionView.EmptyView>
                <ContentView>
                     <ActivityIndicator Color="Chocolate" IsRunning="True" IsVisible="True" />
                </ContentView>
            </CollectionView.EmptyView>

            <CollectionView.ItemTemplate>
                <DataTemplate x:DataType="models:CertificateSigne">
                    <Grid RowDefinitions="*" ColumnDefinitions="*,*,*" ColumnSpacing="6">
                        <VisualStateManager.VisualStateGroups>
                            <VisualStateGroup Name="CommonStates">
                                <VisualState Name="Normal" />
                                <VisualState Name="Selected">
                                    <VisualState.Setters>
                                        <Setter Property="BackgroundColor" Value="Chocolate" />
                                    </VisualState.Setters>
                                </VisualState>
                            </VisualStateGroup>
                        </VisualStateManager.VisualStateGroups>
                        <Label Grid.Row="0" Grid.Column="0" Text="{Binding Certificado}" HorizontalTextAlignment="Start" />
                        <Label Grid.Row="0" Grid.Column="1" Text="{Binding Emisor}" HorizontalTextAlignment="Center" />
                        <Label Grid.Row="0" Grid.Column="2" Text="{Binding Validez}" HorizontalTextAlignment="End" />
                    </Grid>
                </DataTemplate>
            </CollectionView.ItemTemplate>
        </CollectionView>
```

### Version with bug

Release Candidate 3 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 11

**Comment by v-longmin** [comment]
Verified repro on Android 12.0, not repro on windows with VS 17.3.0 Preview 1.0 [32427.505.main]. Repro with above code.

**Comment by xxjnaxx** [comment]
For me EmptyView is working, but only once. See #7076.

**Comment by PureWeen** [comment]
Closing as duplicate of https://github.com/dotnet/maui/issues/7076
